core/capabilities/compute/compute.go

```diff
@@ -245,6 +255,13 @@ func (c *Compute) createFetcher(workflowID, workflowExecutionID string) func(ctx
 		if err != nil {
 			return nil, fmt.Errorf("failed to unmarshal fetch response: %w", err)
 		}
+		if response.StatusCode != http.StatusOK {
```

Comment:
This means that any code in the 200 range that isn't exactly 200 would fail here; I don't think that's the intent.

Reply:
Can we assert on the range instead?

Reply:
addressed.